conductr_cli/sandbox.py

```diff
@@ -0,0 +1,115 @@
+import argcomplete
+import argparse
+import os
+from conductr_cli import sandbox_start, sandbox_stop
+
+DEFAULT_IP = os.getenv('CONDUCTR_IP', '127.0.0.1')
+
+def add_default_arguments(sub_parser):
+    sub_parser.add_argument('-i', '--ip',
+                            default=DEFAULT_IP,
+                            help='ConductR IP address. Defaults to $CONDUCTR_IP or 127.0.0.1.')
+
+def build_parser():
+    # Main argument parser
+    parser = argparse.ArgumentParser('sandbox')
+    subparsers = parser.add_subparsers(title='commands', help='Use one of the following sub commands')
+
+    # Sub-parser for `run` sub-command
+    run_parser = subparsers.add_parser('run',
+                                       help='Run ConductR sandbox cluster',
+                                       usage='%(prog)s [ARGS]',
+                                       formatter_class=argparse.RawTextHelpFormatter)
+    add_default_arguments(run_parser)
+    run_parser.add_argument('-r', '--conductr-role',
+                            dest='conductr_roles',
+                            action='append',
+                            nargs='*',
+                            default=[],
+                            help='Set additional roles allowed by each ConductR node. Defaults to [].',
+                            metavar='')
+    run_parser.add_argument('-d', '--debug-port',
+                            type=int,
+                            default=5005,
+                            help='Debug port to be made public by each of the ConductR containers. Defaults to 5005.',
+                            metavar='')
+    run_parser.add_argument('-e', '--env',
+                            dest='envs',
+                            action='append',
+                            default=[],
+                            help='Set additional environment variables for each ConductR container. Defaults to [].',
+                            metavar='')
+    run_parser.add_argument('-m', '--image',
+                            default='typesafe-docker-registry-for-subscribers-only.bintray.io/conductr/conductr-dev',
+                            help='Docker image to use.\n'
+                                 'Defaults to `typesafe-docker-registry-for-subscribers-only.bintray.io/conductr/conductr`.',
+                            metavar='')
+    run_parser.add_argument('-v', '--image-version',
+                            default='1.0.11',
+                            help='Version of the Docker image to use.\n'
+                                 'Defaults to 1.0.11.\n'
```

Comment:
1.0.12 is released, and the Docker image for it should be available in less than 24 hours. Meanwhile, I suggest changing this now, and tagging a 1.0.12 in your local registry.